akka-docs/src/test/scala/docs/persistence/PersistencePluginDocSpec.scala

```diff
@@ -47,6 +47,16 @@ object PersistencePluginDocSpec {
       //#native-config
       akka.persistence.journal.leveldb.native = off
       //#native-config
+
+      //#compaction-intervals-config
+      # Number of deleted messages per persistence id that will trigger journal compaction
+      akka.persistence.journal.leveldb.compaction-intervals {
+        # persistence-id-1 = interval-1
+        # persistence-id-2 = interval-2
+        # ...
+        # persistence-id-N = interval-N
```

Comment:
Configuring this per-`persistenceId` is pretty low-level... though I guess we could allow more broad-brush configuration by supporting wildcards here later.

Reply:
I agree with @raboof. Would be better with one single property for all persistenceIds. I don't understand your reasoning. The compaction is triggered by a delete, so the persistenceId to is know.

Reply:
all persistenceIds are typically not know up front, i.e. it would be difficult to configure them like this

Reply:
@patriknw when I initially thought of the wildcard feature, I got the perspective of doing a single compaction across all persistence ids, which would have led to the issues described above. I now get that the suggestion was to just use the wildcard to dynamically apply the same interval to perform separate compactions for each persistence id that has not been specifically defined in the configuration. I will proceed with fixing this right away

Reply:
good, I don't think we need to be able to specify different values for different persistenceIds at all, one single property `compaction-interval` would be enough, unless you have a specific need for more fine grained configuration yourself

Reply:
The use case I dealt with at work actually requires a more fine grained configuration, so I would prefer to keep that, if that is not an issue with you.

Reply:
that is fine, thanks for contributing